worldengine/astar.py

```diff
@@ -198,14 +197,6 @@ def _handle_node(self, x, y, from_node, destination_x, destination_y):
         return None
 
 
-def _matrix_to_array(matrix):
```

Comment:
I just profiled a<b<c to gain about a millisecond while this change gained about 25% of the total runtime. :D I shouldn't try to look into those super-small things anymore. :P